src/codec_aom.c

```diff
@@ -464,6 +464,9 @@ static avifBool avifProcessAOMOptionsPostInit(avifCodec * codec, avifBool alpha)
                                   aom_codec_error_detail(&codec->internal->encoder));
             return AVIF_FALSE;
         }
+        if (!strcmp(key, "cq-level")) {
+            codec->internal->cqLevelSet = AVIF_TRUE;
+        }
```

Comment:
Right now we only set the `cqLevelSet` boolean correctly at lines 501-503 below, when libaom does not have the `aom_codec_set_option()` function. I don't know why I missed this spot.

Since libaom v3.0.0 or later has the `aom_codec_set_option()` function, this bug affects all recent versions of libaom.